### DISCUSSION TITLE
LED brightness in VPX

### DISCUSSION THREAD

**Discussion by 4starpizza** [missionpinball/missionpinball#31] [Q&A]
I'm getting started with MPF and using the mpf-vpcom-bridge to interface with a VPX table to do my yaml'ing before starting to build my physical machine. This has been a productive setup and I've learned a ton. 

One limitation I've encountered is that the VPX platform definition in MPF (platforms/virtual_pinball) only transmits light brightness as a boolean state. This is coded in virtual_pinball.py. I've changed things locally to send a floating point brightness to VPX instead with success. I'm considering contributing this enhancement back to the repo.  I could add a new method to the VirtualPinballPlatform class so that the existing method still works as-is using booleans. Any thoughts?

**Comment by borgdog** [comment]
Sounds like a nice improvement, certainly wouldn't want to break any existing stuff though.

**Comment by 4starpizza** [comment]
PR submitted for this: https://github.com/missionpinball/mpf/pull/1666
